README.md

```diff
@@ -90,14 +80,7 @@ $ CREATE SCHEMA exports AUTHORIZATION hot;
 $ sudo apt-get install software-properties-common
 $ sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable
 $ sudo apt-get update
-$ sudo apt-get install gdal-bin libgdal-dev
-</pre>
-
-To install the python GDAL bindings into your virtualenv you need to tell pip where to find the libgdal header files, so in your shell run:
-
-<pre>
-$ export CPLUS_INCLUDE_PATH=/usr/include/gdal
-$ export C_INCLUDE_PATH=/usr/include/gdal
+$ sudo apt-get install gdal-bin libgdal-dev python-gdal
```

Comment:
ah, so that's why you need to use `system-site-packages`, it does make it a bit easier...